### PR TITLE
update meetup leadership

### PR DESCRIPTION
**Pull request by mjang**

### CODE REVIEW

**Comment by mjang** [comment]
DON'T MERGE until people in #meta-meetup-orgs approve

File: docs/team.rst [hunk 1]
```diff
@@ -79,9 +79,12 @@ Roles
...
 
-* `Mike Jang <https://twitter.com/TheMikeJang>`__, Senior Staff Technical Writer, ForgeRock *
-* `Margaret Eker <https://twitter.com/meker>`_
-* `Rose Williams <https://twitter.com/ZelWms>`_
+* `Rose Williams <https://twitter.com/ZelWms>`_ *
+* `Margaret Eker <https://twitter.com/meker>`_ *
+* `Mike Jang <https://twitter.com/TheMikeJang>`_ *
+* `Marybeth Alexander <https://twitter.com/knowledge_owl>`_
+* `Catherine Heath <https://twitter.com/awaywithwords2>`_
+* `Swapnil Ogale <https://twitter.com/swapnilogale>`_
```
Comment (ericholscher) [inline docs/team.rst]: I wonder if we should split this up into regions on this page? So people know who to contact about which region (UTC-8, UTC, UTC+8 unless we have a better grouping?)

File: docs/team.rst [hunk 2]
```diff
@@ -79,9 +79,12 @@ Roles
 Members
 ```````
 
-* `Mike Jang <https://twitter.com/TheMikeJang>`__, Senior Staff Technical Writer, ForgeRock *
-* `Margaret Eker <https://twitter.com/meker>`_
-* `Rose Williams <https://twitter.com/ZelWms>`_
+* `Rose Williams <https://twitter.com/ZelWms>`_ *
+* `Margaret Eker <https://twitter.com/meker>`_ *
```
Comment (ericholscher) [inline docs/team.rst:83]: Is Margaret still involved? I thought she stepped down.

**Review by ericholscher — COMMENTED** [review]
Glad to see this getting updated :)

File: docs/team.rst (hunk 2 above)
Comment (mjang) [reply docs/team.rst:83]: Margaret ping'd me after the conference, saying that she felt re-energized, and was ready to take more leadership. I figure between Margaret and Rose, I will be able to step back!

I've invited you to our meta-meetup-orgs Trello / Slack.

File: docs/team.rst (hunk 1 above)
Comment (mjang) [reply docs/team.rst]: Sounds reasonable.

File: docs/team.rst (hunk 2 above)
Comment (mjang) [reply docs/team.rst:83]: And with the innovations I've seen from Austin in the past year, I think she's well qualified to spread the word on -- (should I call it?) "non-traditional" meetup formats, cc @rosewms

**Review by meker12 — APPROVED** [review]

**Review by ericholscher — APPROVED** [review]
👍

**Comment by mjang** [comment]
Note: I've removed MaryBeth Alexander from the list, since I haven't heard from her lately. I'll be happy to add her in as soon as I get her OK to do so. 

Also @plaindocs , I'm not sure about the build failure. If the failure is "no big deal", you're welcome to merge.

**Pull request merged**